web/src/message_fetch.js

```diff
@@ -249,16 +206,16 @@ export function load_messages(opts, attempt = 1) {
     //   data.narrow = opts.msg_list.data.filter.public_terms()
     //
     // But support for the all_messages_data sharing of data with
-    // message_lists.home and the (hacky) page_params.narrow feature
+    // all messages view and the (hacky) page_params.narrow feature
```

Comment:
We're renaming "all messages" view, so maybe it's worth writing these as "the combined feed view" just to avoid having to change the comments an extra time?

Reply:
Done